src/asyncBufferedTransformer.ts

```diff
@@ -23,16 +23,24 @@ export type AsyncBufferedTransformerOptions = {
 };
 
 export async function* asyncBufferedTransformer<T>(
-  stream: AsyncIterable<PromiseWrapper<T>>,
+  stream: Iterable<PromiseWrapper<T>> | AsyncIterable<PromiseWrapper<T>>,
   { numberOfParallelExecutions }: AsyncBufferedTransformerOptions,
   errorLogger: (message: string, ...params: any) => void = console.log
 ): AsyncIterable<T> {
-  if (numberOfParallelExecutions < 2) {
+  if (numberOfParallelExecutions < 0) {
     throw new Error(
-      "numberOfParallelExecutions, otherwise there is no parallel execution"
+      `numberOfParallelExecutions was ${numberOfParallelExecutions}, expected >= 0`
```

Comment:
Please change it in the comment in README.md as well (`// must be >= 2 for the parallel execution to make sense (otherwise throws an Error)`)

Reply:
done.